src/index.html

```diff
@@ -19,9 +19,423 @@
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#smartphones"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                data-qa="nav-hover"
+                href="#laptops"
+                class="nav__link"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#gadgets"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#tablets"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#photo"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#video"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="products">
+      <div class="products__wrapper">
+        <div
+          class="products__card card"
+          data-qa="card"
+        >
+          <div class="card__container">
+            <div class="card__image">
+              <img
+                src="images/imac.jpeg"
+                alt="APPLE A1419 iMac 27"
+                class="card__img"
+              />
+            </div>
+            <div class="card__name">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p class="card__product-code">Product code: 195434</p>
+            </div>
+            <div class="card__review-block">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__review-text">Reviews: 5</p>
+            </div>
+            <div class="card__price-block">
+              <p class="card__price-text">Price:</p>
+              <p class="card__price">$2,199</p>
+            </div>
+            <div class="card__button button">
+              <a
+                href="#"
+                class="button__link"
+                data-qa="card-hover"
+              >
+                BUY
+              </a>
+            </div>
+          </div>
+        </div>
```

Comment:
I see you are repeating the same block of code for the product card multiple times. This is not a good practice as it leads to code duplication. Consider using a templating engine or a JS framework to dynamically generate these cards.